app/services/green_lanes/fetch_goods_nomenclature_service.rb

```diff
@@ -54,7 +54,7 @@ def call
              .order(Sequel[:goods_nomenclatures][:producline_suffix], Sequel[:goods_nomenclature_indents][:number_indents])
```

Comment:
I know not related to this PR but would be good to extract some of the code out of the call method to improve readability and understanding.